webapp/src/utils.ts

```diff
@@ -132,3 +132,40 @@ export function runCallsSlashCommand(command: string, channelId: string, teamId:
         },
     }, window.origin);
 }
+
+export function safeJSONParse<T>(value: unknown): T | null {
+    if (!value || typeof value !== 'string') {
```

Comment:
I think this would be enough:

```suggestion
    if (typeof value !== 'string') {
```

We can let the case of empty string being handled by `JSON.parse`.